spec/classes/collectd_plugin_mysql_database_spec.rb

```diff
@@ -84,6 +84,13 @@
           facts.merge(collectd_version: '5.6')
         end
 
+        let :params do
+          {
```

Comment:
This should be in a different context that shows what happens when host and port are specified as they were before (localhost/3306).

Reply:
You are obviously correct; fixing :-)